.env.example

```diff
@@ -28,6 +28,7 @@ EMBEDLY_KEY=**
 GALAXY_TOKEN=**
 GOOGLE_CSE_CX=**
 GOOGLE_CSE_KEY=**
+VORTEX_TOKEN=**
```

Comment:
I just now realise that this `**` placeholder will make it pass the `mustHave` check incorrectly. Perhaps we should just not have placeholder values?